appeals/api/src/server/config/schema.js

```diff
@@ -1,68 +1,75 @@
 import joi from 'joi';
 
-export default joi.object({
-	NODE_ENV: joi.string().valid('development', 'production', 'test'),
-	PORT: joi.number(),
-	SWAGGER_JSON_DIR: joi.string(),
-	DATABASE_URL: joi.string(),
-	DISABLE_INTEGRATIONS: joi.boolean(),
-	defaultApiVersion: joi.string(),
-	serviceBusOptions: joi.object({
-		hostname: joi.string().optional()
-	}),
-	msal: joi.object({
-		clientId: joi.string().optional(),
-		clientSecret: joi.string().optional(),
-		tenantId: joi.string().optional()
-	}),
-	log: joi.object({
-		levelFile: joi.string(),
-		levelStdOut: joi.string()
-	}),
-	cwd: joi.string(),
-	featureFlags: joi.object().pattern(/featureFlagBoas\d+[A-Za-z]+/, joi.boolean()),
-	serviceBusEnabled: joi.boolean().optional(),
-	timetable: joi.object({
-		FPA: joi.object({
-			lpaQuestionnaireDueDate: joi.object({
-				daysFromStartDate: joi.number()
-			}),
-			statementReviewDate: joi.object({
-				daysFromStartDate: joi.number()
+export default joi
+	.object({
+		NODE_ENV: joi.string().valid('development', 'production', 'test'),
+		PORT: joi.number(),
+		SWAGGER_JSON_DIR: joi.string(),
+		DATABASE_URL: joi.string(),
+		DISABLE_INTEGRATIONS: joi.boolean(),
+		defaultApiVersion: joi.string(),
+		serviceBusOptions: joi.object({
+			hostname: joi.string().optional()
+		}),
+		msal: joi.object({
+			clientId: joi.string().optional(),
+			clientSecret: joi.string().optional(),
+			tenantId: joi.string().optional()
+		}),
+		log: joi.object({
+			levelFile: joi.string(),
+			levelStdOut: joi.string()
+		}),
+		cwd: joi.string(),
+		featureFlags: joi.object().pattern(/featureFlagBoas\d+[A-Za-z]+/, joi.boolean()),
+		serviceBusEnabled: joi.boolean().optional(),
+		timetable: joi.object({
+			FPA: joi.object({
+				lpaQuestionnaireDueDate: joi.object({
+					daysFromStartDate: joi.number()
+				}),
+				statementReviewDate: joi.object({
+					daysFromStartDate: joi.number()
+				}),
+				finalCommentReviewDate: joi.object({
+					daysFromStartDate: joi.number()
+				})
 			}),
-			finalCommentReviewDate: joi.object({
-				daysFromStartDate: joi.number()
+			HAS: joi.object({
+				lpaQuestionnaireDueDate: joi.object({
+					daysFromStartDate: joi.number()
+				})
 			})
 		}),
-		HAS: joi.object({
-			lpaQuestionnaireDueDate: joi.object({
-				daysFromStartDate: joi.number()
+		govNotify: joi
+			.object({
+				api: joi.object({
+					key: joi.string() // optional if not NODE_ENV=production
+				}),
+				template: joi.object({
+					validAppellantCase: joi.object({
+						id: joi.string()
+					})
+				}),
+				testMailbox: joi.string()
```

Comment:
For completeness perhaps `testMailbox` could be optional when not production as with `api.key` as the two are related but it also works fine as it is too.

Reply:
That's fair, the reason I left if its that testMailbox has a default even if TEST_MAILBOX isn't set - so it'll always pass validation.